test/integration/testutil/testutil.go

```diff
@@ -66,8 +66,10 @@ func CreateWebServices() []*restful.WebService {
 	addRoutes(w, buildRouteForType(w, "uniontype", "InlinedUnion")...)
 	addRoutes(w, buildRouteForType(w, "custom", "Bal")...)
 	addRoutes(w, buildRouteForType(w, "custom", "Bak")...)
-	addRoutes(w, buildRouteForType(w, "custom", "Bac")...)
-	addRoutes(w, buildRouteForType(w, "custom", "Bah")...)
+	if includeV2SchemaAnnotation {
```

Comment:
All other tests use true for the new flag. OpenAPI v2->v3 conversion tests have this disabled because they include the x-kubernetes-v2-schema extension and there is no way to provide equality with the generated v3 schema and upsampled v2->v3.

Reply:
I don't understand your comment here, or what is special about the "Bac" and "Bah" routes that they would need to be excluded for new conversion tests. Could you provide some more context?

Also, I am told by @apelisse the `x-kubernetes-v2-schema` extension is being removed so maybe this does not matter any longer?

Reply:
```
   "custom.Bah": {
     "type": "object",
     "x-kubernetes-v2-schema": {
      "type": "test-type",
      "format": "test-format"
     }
    }
```

OpenAPI V2 will interpret this with "test-type" and OpenAPI V3 will interpret this as "object". Converting OpenAPI V2 to V3 will not magically change the type and will cause a type mismatch in our comparison. This mismatch is intended because V2 to V3 is a lossy conversion but will cause the tests to fail.